src/Swarm/TUI/View.hs

```diff
@@ -793,20 +797,29 @@ drawKeyMenu s =
     [ ("←↓↑→ / hjkl", "scroll") | creative
     ]
       ++ [("c", "recenter") | not viewingBase]
+      ++ [("f", "FPS")]
   keyCmdsFor (Just RobotPanel) =
-    [ ("Ret", "focus")
+    [ ("Ret", "pop out")
```

Comment:
We have enough space for clearer message:
```suggestion
    [ ("Enter", "pop out entity description")
```

Reply:
Depends on how big the terminal is. :thinking: I wonder if we could make things "responsive" so e.g. it uses abbreviations, shorter messages,  etc when there's not much space.

Reply:
I think I'll stick with the shorter `pop out` for now.  These are supposed to be small hints to remind you what the keys do, not full descriptions, and I'd rather not have one be that long.  However, changing `Ret` to `Enter` is a good idea.